taskpools/chase_lev_deques.nim

```diff
@@ -148,6 +150,10 @@ proc push*[T](deque: var ChaseLevDeque[T], item: T) =
     # Full queue
     deque.grow(a, t, b)
 
+  if not deque.garbage.isNil and b == t:
```

Comment:
```suggestion
  elif b-t == 0:
```

`garbageCollect` already contains the `dequeue.garbage != nil != nil` check, implicitly.